chapter_appendix_math/geometry-linear-algebric-ops_vn.md

```diff
@@ -1447,7 +1454,9 @@ we can consider the Einstein summation seen above
 and strip out the indices themselves to get the implementation:
 -->
 
-*dịch đoạn phía trên*
+Tổng Einstein đã được cài đặt thông qua hàm ```np.einsum```.
```

Comment:
Summation là phép tính tổng nhé, tổng đơn thuần là sum thôi.
```suggestion
Phép tính tổng Einstein đã được lập trình thông qua hàm ```np.einsum```.
```

Reply:
@lkhphuc trong CNTT thì "implement" là "cài đặt" đúng rồi nhỉ

Reply:
@rootonchair "implement" là "triển khai", cài đặt là "install".

Reply:
@tiepvupsu hmm. Lạ nhỉ. Trường em lại toàn dùng "cài đặt". 
"implement a function/class" -> "Cài đặt một hàm/lớp"

Reply:
Tạm thời em sẽ dịch "implement" theo suggestion